### DISCUSSION TITLE
Editing e-mailers

### DISCUSSION THREAD

**Discussion by Noeltim** [forem/forem#15972] [Feature Requests]
Challenge : Editing current emailers such as Confirmation Email upon registration, Password Reset emailers etc. 

Workaround: Fork the repo and edit the emailers from this repo https://developers.forem.com/backend/emails

Best case scenario: Easy way to edit from the config dashboard in the forem.

**Comment by ellativity** [comment]
Yes! Community success agrees with this. (https://github.com/forem/rfcs/issues/324 for internal reference)
